<!DOCTYPE html>
<html lang="en">
<head>
<title>BRC</title>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<link href="https://fonts.googleapis.com/css2?family=Allura&display=swap" rel="stylesheet">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">

<style>
* {
  box-sizing: border-box;
}

/* Style the body */
body {
  margin: 0;

  background-color: #DFD8CA;

}

/* Header/logo Title */



/* Style the top navigation bar */
.navbar {
  overflow: hidden;
  background-color: #333;
}

/* Style the navigation bar links */
.navbar a {
  float: left;
  display: block;
  color: white;
  text-align: center;
  padding: 14px 20px;
  text-decoration: none;
  font-size: 18px;
  font-family: Calibri; }

/* Right-aligned link */
.navbar a.right {
  float: right;
}

/* Change color on hover */
.navbar a:hover {
  background-color: #ddd;
  color: black;
}
.navbar .icon {
  display: none;
}
@media screen and (max-width: 700px) {
  .navbar a:not(:first-child) {display: none;}
  .navbar a.icon {
    float: right;
    display: block;
  }
}

@media screen and (max-width: 700px) {
  .navbar.responsive {position: relative;}
  .navbar.responsive .icon {
    position: absolute;
    right: 0;
    top: 0;
  }
  .navbar.responsive a {
    float: none;
    display: block;
    text-align: left;
  }
}


/* Column container */
.row {  
  display: -ms-flexbox; /* IE10 */
  display: flex;
  -ms-flex-wrap: wrap; /* IE10 */
  flex-wrap: wrap;
  font-family: Calibri;   
}

/* Main column */


/* Fake image, just for this example */




.footer {
  padding: 10px;
  text-align: center;
  background: #105652;
  color: white;
  font-family: Calibri; 
}
.footer a{
  text-decoration: none; 
  color: white;
}


/* Responsive layout - when the screen is less than 700px wide, make the two columns stack on top of each other instead of next to each other */
@media screen and (max-width: 700px) {
  .row {   
    flex-direction: column;
  }
}

/* Responsive layout - when the screen is less than 400px wide, make the navigation links stack on top of each other instead of next to each other */
@media screen and (max-width: 400px) {
  .navbar a {
    float: none;
    width: 100%;
  }
}

.header {
  font-family: Calibri;   padding: 20px;
  font-size: 20px;
  text-align: center;
  background: #105652;
  color:white;
}
.rightcolumn {
  float: left;
  width: 25%;
  padding-left: 20px;
}
.leftcolumn {   
  float: left;
  width: 75%;
}
h4 {
  font-family: Calibri; 
}
.card {
   background-color: white;
   padding: 20px;
   margin-top: 20px;
}
.card1{
    background-color: #105652;

    color:white;
   padding: 20px;
   margin-top: 30px;
}
.name {
  background-color: white;
  color:black;
  width: 100%;
  padding: 20px;
}
.card1 a{
    text-decoration: none;
   color:white;
}
.card1 a:hover {
  color: #FFCC1D;
}

/* Clear floats after the columns */
.row:after {
  content: "";
  display: table;
  clear: both;
}



/* Responsive layout - when the screen is less than 800px wide, make the two columns stack on top of each other instead of next to each other */
@media screen and (max-width: 800px) {
  .leftcolumn, .rightcolumn {   
    width: 100%;
    padding: 0;
  }
}
</style>
</head>
<body>


<div class="navbar" id="myTopnav">
    <a href="../index.html">Home</a>

  <a href="../blog.html">Blog</a>
  <a href="../initiative.html">Initiative</a>
  <a href="../newsletter.html">News Letter</a>
 
  <a href="../about.html" class="right">About</a>
  <a href="javascript:void(0);" class="icon" onclick="myFunction()">
    <i class="fa fa-bars"></i>
  </a>
</div>

<div class="header">
  <h2>Unravelling Retroactivity: An Enigma </h2>
</div>
<div class="row">
  <div class="leftcolumn" style="padding: 10px;">
    <div class="card">
     
     
      <h4>Introduction</h4>
      
      <p>Today, synthetic biology has grown leaps and bounds and transformed with applications and approaches inspired by electronics and software design. For instance, the amplified bio-sensing, timed genetic circuits, oscillators and switches. Modules in a biological system can be thought of as a network of reactions. Our goal is to modularize, device standardized BioBricks such as coding sequences, promoters, ribosomal binding sites and terminators to easily design, assemble pathways and engineer biological circuits to perform novel tasks, such as to achieve a predictable output behaviour with a defined input. 
        </p>
        <img src='../images/blog/blog 3 fig 1.png' height="100px" >

        <p>Just like in analogue and hydraulic systems, impedance or load like effects can arise in a biological system on the interconnection of components. This phenomenon is called &lsquo;retroactivity&rsquo;. When the modules are interconnected, retroactivity is characterized by signals from downstream modules (that receive the arriving signals) to upstream modules (that send the original signals).&nbsp;</p>
        <p>Nonetheless, an electrical engineer&rsquo;s life is not as complicated as a systems biologist! An electrical engineer does not have to worry about the components changing dynamically, for example, the dimensions of capacitors and resistors are almost constant, and lines of code do not rewrite themselves, but in a genetic circuit, random mutations can change everything! Besides, the solid-state transistors can be explained using simple Boolean logic, whereas computing the functionality of transcription factors using Boolean logic is a challenging task. Furthermore, electric circuits are modular, hard-wired, and the energy-loss and efficiency well-estimated, but a biological circuit operates in a diffusive environment and the molecular cross-talk cannot be ignored. It is also difficult to measure the leaky expression of the system. Moreover, an electrical engineer commonly deals with nano-second transitions in logic circuits, whereas a systems biologist needs to study gene-regulatory circuits that may operate on minutes to hours&rsquo; scale.&nbsp;</p>
        <p>In other words, a systems biologist has to deal with all the perplexity and complexity!&nbsp;</p>
        <img src='../images/blog/blog 3 fig 2.png' height="150px">
        <img src='../images/blog/blog 3 fig 3.png' height="150px" >

        <p>Let us understand retro-activity with the above diagram.&nbsp;</p>
        <p>The function of module A is to produce a finite set of &lsquo;n&rsquo; molecules when an input signal is given. However, a decrease in the output of module A by 'm' molecules is observed when connected to second module B. These &lsquo;m&rsquo; molecules could have been degraded, covalently modified or sequestered thereby decreasing the output of A.&nbsp;</p>
        <p>Therefore, it is evident that retroactivity is a property of both natural and synthetic systems and depending on the strength, the downstream molecules can significantly change or even disrupt the functionality of the upstream molecules.&nbsp;</p>
        <p>To enhance our understanding of retroactivity, it is essential to characterise modules. <strong>What is a module? </strong>As per the dictionary, a module is &lsquo;an independent unit that can be used to construct more complex structures.&rsquo;&nbsp;</p>
        <p>In a circuit, it becomes essential to characterize the functional modules. These are composed of different kinds of interacting molecules and perform specific, discrete functions that cannot be easily predicted from studying the components in isolation. Hartwell (1999) defines a functional module as &ldquo;features that act together in performing some discrete physiological function are semi-autonomous in relation to other functional modules&rdquo;.&nbsp;</p>
        <p>The next apparent question that arises is - <strong>Are biological circuits truly modular? </strong>We can say that at a fundamental level, there is a modular composition in biological systems consisting of genes, operons and horizontal transfer elements. But, the constant inter-linkage between parts (such as covalent modification of enzymes and transcription factors), participation in multiple pathways and intertwining of functions makes it a difficult task to establish the structural and functional boundaries that delimit modules.&nbsp;</p>
        <p>Thus, in a biological system, there is a balance between modularity and connectivity. Hence, we can conclude that &lsquo;biological systems are virtual units, in other words, QUASI-MODULAR!&rsquo;&nbsp;</p>
        <p>Structures, molecules and complexes are shared across different pathways. The limitations that lead to autonomy are highly dynamic and can change the structure at any given time. How biochemical modules can be demarcated is still an unanswered question and is being studied. Some approaches exist based on network-clustering methods applied to experimental data (Rives &amp; Galitski, 2003), and for developing a theoretical framework for the analysis of modular networks (Bruggeman, Westerhoff, Hoek, &amp; Kholodenko, 2002). Three biologically motivating criteria for defining functional units are: (1) common physiological task, (2) common genetic units and (3) common signal transduction network. Saez-Rodriguez et al. (2005) introduced retroactivity in biological systems and proposed that modules in cell signal transduction networks can be established based on minimal retroactivity interactions.&nbsp;</p>
        <p><strong>Why is it important to consider retroactivity? </strong>It is crucial for a deeper understanding of the interconnection between the functional modules and their impact on the overall behaviour of the cell. This influences the need for novel experimental techniques and theoretical frameworks to formally characterize and quantify effects of retroactivity. Further, retroactivity also provides an opportunity to utilize mathematical and computational models. These may unravel the properties of previously studied motifs and patterns of interconnection; the capacity to minimize or increase retroactivity.&nbsp;</p>
        <p><strong>How does retroactivity manifest in a living cell in the first place? </strong>Transcription factors may drive too many target promoters, or nucleosomes could restrict the binding of a transcription factor to its target. Due to squelching and enzyme degradation, large amounts of regulatory elements may deplete the cell&rsquo;s resources, leading to a reduction of the system&rsquo;s performance. Further, there may be &ldquo;non- functional&rdquo; transcription factor-binding sites which can act as loads. The Cascades of covalent modification cycles, methylation&ndash;demethylation, activation&ndash;inactivation of GTP-binding proteins, phosphorylation&ndash; dephosphorylation manifest in several regulations. So far, it seems as though retroactivity is bad and is reducing the performance of the system. But in certainty, retroactivity can have important functional roles in living systems!&nbsp;</p>
        <p>Retroactivity plays an important role in integrating information into the central nervous system from different senses and dictates the organism's behaviour. The generation of action potential and regulation of neurotransmitters in nerve cells also display retroactivity. Retroactivity can additionally explain the evolution of certain proteins such as histones, actin and tubulin, that interact with many others and have altered very little over time. This can be due to the very reason for their participation in many different interactions. The decision-making module in bacteriophage lambda from lytic to lysogenic cycles is also a classic example of retroactivity. Likewise, the regulation of our fantastic immune system and the cross-talk between its various players is an excellent example!&nbsp;</p>
        <p>Finally, let us review the pros and cons of retroactivity meant for a system in synthetic biology. Retroactivity can cause delays and disrupt systems. While designing a gene circuit, the goal is to ensure that it is robust to perturbations imposed by constraints on its design. However, only a small fraction of the possible circuits formed by interconnection can execute a desired function. According to Alexander et al. (2009) &ldquo;knowledge of network structure is often not sufficient to infer function, and dynamic modularity can exist in the absence of structural modularity.&rdquo; Therefore, shaping a biosynthetic system in a modular way is a complicated task.&nbsp;</p>
        <p>Modularity promotes robustness to component tolerances and allows reconfiguration for new conditions and the capacity to test functions. It improves the organization of mechanisms addressing local problems in a network. However, retroactivity has a homeostatic role, which is important for living systems. Retroactivity transfers the system from an ultrasensitive response regime to a lower sensitivity regime. It is clear that retroactivity has a functional role and is not just a nuisance signal from its manifestation in living beings.&nbsp;</p>
        <p>Whether biological systems tend to minimize or potentiate retroactivity is a matter of debate in the current scenario. Quasi-modularity can be a survival strategy because preserving functions under diverse settings is imperative for life. Therefore, to fully realize the potential and promise of synthetic biology, and understand the comprehensive processes we must appreciate and consider the complexity and specificity!&nbsp;</p>
  
</div>
 
    </div>
    <div class="rightcolumn">
      <div class="card1">
        <h2>Written by</h2>
        <div class="name" style="height:100px;">Roshni Shetty</div>
       
      </div>
      <div class="card1">
        <h3>Reference</h3>
        <div class="name"><p>Pantoja-Hern&aacute;ndez L and Mart&iacute;nez-Garc&iacute;a JC (2015) Retroactivity in the context of modularly structured biomolecular systems. Front. Bioeng. Biotechnol. 3:85. doi: <a href="https://doi.org/10.3389/fbioe.2015.00085" style="color: purple;">10.3389/fbioe.2015.00085</a></p></div><br>
        
      </div>
      <div class="card1">
        <h3>Share:<a href='http://twitter.com/share?text=Synthetic%20Biology%20Goes%20Cell-Free&url=https://biotechstudentassociation.github.io/SyntheticBiologyGoesCellFree/'>Twitter</a>,<a href='https://www.facebook.com/sharer.php?u=https://biotechstudentassociation.github.io/SyntheticBiologyGoesCellFree/'>Facebook</a></h3>
        
      </div>
    </div>
  </div>
  <br>
<div class="footer">
  <p>Follow Us &emsp; <a href="https://www.instagram.com/bsaiitm/" target="_blank" ><i class="fa fa-instagram" style="font-size:36px"></i></a>&emsp;<a href="https://www.facebook.com/biotechresearchclubiitmadras" target="_blank" ><i class="fa fa-facebook-square" style="font-size:36px"></i></a>&emsp;<a href="mailto:biotechresearchclubiitm@gmail.com" target="_blank"><i class="fa fa-envelope" style="font-size:36px"></i></a>&emsp;<a href="https://chat.whatsapp.com/AnLJuBtdjXP5ltT8dtlLrR" target="_blank"><i class="fa fa-whatsapp" style="font-size:36px"></i></a></p>
 
</div>
<script src="../navbar.js">
    
</script>


</body>
</html>
