<!DOCTYPE html>
<html lang="en">
<head>
<title>Page Title</title>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<link href="https://fonts.googleapis.com/css2?family=Allura&display=swap" rel="stylesheet">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">

<style>
* {
  box-sizing: border-box;
}

/* Style the body */
body {
  margin: 0;

  background-color: #DFD8CA;

}

/* Header/logo Title */



/* Style the top navigation bar */
.navbar {
  overflow: hidden;
  background-color: #333;
}

/* Style the navigation bar links */
.navbar a {
  float: left;
  display: block;
  color: white;
  text-align: center;
  padding: 14px 20px;
  text-decoration: none;
  font-size: 18px;
  font-family: Calibri; }

/* Right-aligned link */
.navbar a.right {
  float: right;
}

/* Change color on hover */
.navbar a:hover {
  background-color: #ddd;
  color: black;
}
.navbar .icon {
  display: none;
}
@media screen and (max-width: 700px) {
  .navbar a:not(:first-child) {display: none;}
  .navbar a.icon {
    float: right;
    display: block;
  }
}

@media screen and (max-width: 700px) {
  .navbar.responsive {position: relative;}
  .navbar.responsive .icon {
    position: absolute;
    right: 0;
    top: 0;
  }
  .navbar.responsive a {
    float: none;
    display: block;
    text-align: left;
  }
}


/* Column container */
.row {  
  display: -ms-flexbox; /* IE10 */
  display: flex;
  -ms-flex-wrap: wrap; /* IE10 */
  flex-wrap: wrap;
  font-family: Calibri;   
}

/* Main column */


/* Fake image, just for this example */




.footer {
  padding: 10px;
  text-align: center;
  background: #105652;
  color: white;
}
.footer a{
  text-decoration: none; 
  color: white;
}


/* Responsive layout - when the screen is less than 700px wide, make the two columns stack on top of each other instead of next to each other */
@media screen and (max-width: 700px) {
  .row {   
    flex-direction: column;
  }
}

/* Responsive layout - when the screen is less than 400px wide, make the navigation links stack on top of each other instead of next to each other */
@media screen and (max-width: 400px) {
  .navbar a {
    float: none;
    width: 100%;
  }
}

.header {
  font-family: Calibri;   padding: 20px;
  font-size: 20px;
  text-align: center;
  background: #105652;
  color:white;
}
.rightcolumn {
  float: left;
  width: 25%;
  padding-left: 20px;
}
.leftcolumn {   
  float: left;
  width: 75%;
}
h4 {
  font-family: Calibri; 
}
.card {
   background-color: white;
   padding: 20px;
   margin-top: 20px;
}
.card1{
    background-color: #105652;

    color:white;
   padding: 20px;
   margin-top: 30px;
}
.name {
  background-color: white;
  color:black;
  width: 100%;
  padding: 20px;
}
.card1 a{
    text-decoration: none;
   color:white;
}
.card1 a:hover {
  color: #FFCC1D;
}

/* Clear floats after the columns */
.row:after {
  content: "";
  display: table;
  clear: both;
}



/* Responsive layout - when the screen is less than 800px wide, make the two columns stack on top of each other instead of next to each other */
@media screen and (max-width: 800px) {
  .leftcolumn, .rightcolumn {   
    width: 100%;
    padding: 0;
  }
}
</style>
</head>
<body>


<div class="navbar" id="myTopnav">
    <a href="../index.html">Home</a>

  <a href="../blog.html">Blog</a>
  <a href="../initiative.html">Initiative</a>
  <a href="../newsletter.html">News Letter</a>
 
  <a href="../about.html" class="right">About</a>
  <a href="javascript:void(0);" class="icon" onclick="myFunction()">
    <i class="fa fa-bars"></i>
  </a>
</div>

<div class="header">
  <h2>The new frontier with genome engineering with CRISPR-Cas9</h2>
</div>
<div class="row">
  <div class="leftcolumn" style="padding: 10px;">
    <div class="card">
     
     
     
      
        <p>Doudna and Charpentier were the principal investigators in the labs that published their paper detailing CRISPR&rsquo;s adaptability to gene editing in Science way back in 2012. Their work drove the way for a new frontier in genome engineering, revolutionising our perspective on gene editing once and for all. It is only befitting that we take a look at the way they review the impact that CRISPR-Cas9 had on the field.&nbsp;</p>
        <p>Gene editing is not a new and certainly not a novel concept. It has been around since the late &lsquo;70s, when gene replacement was first achieved in yeast. There were many discoveries along the way, including and not limited to the discovery of Zinc Finger Proteins, the discovery of Homology Directed Repair and non-homologous end joining, all new methods and techniques that slightly furthered the case that we might be able to, one day, edit the human genome. Then, in the 1990&rsquo;s, scientists across the world managed to, for the first time, sequence the entire human genomes. Everyone thought that this would change the way we deal with human genetics forever. But none of these discoveries had the impact they&rsquo;d been advertised to have, until now, that is.&nbsp;</p>
        <img src='../images/blog/blog 6 fig 1.png' height="200px" style="margin-left: 25%;">
        <p>Fig 1. The timeline of progress on genome editing (without CRISPR).1</p>
        <p>&nbsp;</p>
        <h1>History of CRISPR-Cas</h1>
        <p>In the 2000&rsquo;s, a range of laboratories began investigating CRISPR for unrelated reasons. They were found in certain bacteria and archaea, which seemed to be clustered repeats of certain genetic sequences. It was later discovered that the sequences seemed viral in origin or resembled certain plasmids. Further down the line, it was proposed that CRISPR-Cas was an adaptive immune response to past invasions, and that the sequences were antisense reminders of invading sequences. The same was established to be true in <em>Streptococcus thermophilus</em>. After this was it first hypothesised that bacteria could be weaponized to act as a deterrent to phages. This was tested successfully in 2008, and CRISPR RNA&rsquo;s were shown to act as guides to locations that had to be spliced from invading genomes.&nbsp;</p>
        <h1>CRISPR-Cas: How does it work?&nbsp;</h1>
        <p>The operation of the CRISPR-Cas system is simple. Let us look into some basics on DNA first. DNA is a polymer comprised of four molecular bases, commonly represented as A, G, T and C. The structure of DNA is based on base-pairing, where A and T pair with each other and G and C pair do likewise. It is this pairing that results in a double stranded structure of DNA, where each strand perfectly binds with the opposing strand<sup>4</sup>. RNA has a structure that is similar to DNA, except for the T being replaced with a U. RNA is <em>transcribed </em>from DNA, and the RNA obtained is almost identical to the primary or the <em>Watson </em>strand of DNA, named after one of the researchers who discovered the structure of DNA. When the RNA is exported out of the nucleus, it is translated into proteins, which drive functions around the body. RNA, in and of itself, has the ability to base pair with a strand of DNA. RNA exists in nature usually as a single stranded molecule.&nbsp;</p>
        <p>Fig. 2. DNA Base binding and complementarity.</p>
        <p>In natural systems, invading virus/plasmid&rsquo;s RNA have been <em>reverse transcribed </em>or reversed into DNA and subsequently integrated into host genomes in the form of repeated sequences. Hence, CRISPR stands for Clustered Regularly Interspaced Short Palindromic Repeats. All such sequences are clustered in a region in the host genome called the CRISPR locus. The Cas protein, prepared by the cellular machinery, and the RNA similar to the viral RNA, that was prepared by the host, form a complex and begin surveillance of the cytoplasm. This is known as the CRISPR-Cas surveillance complex.</p>
        <img src='../images/blog/blog 6 fig 3.png' height="200px"style="margin-left: 25%;"> 
        <p>Figure 3. CRISPR Cas in Nature.<sup>5</sup></p>

        <p>The surveillance complex, now containing the CRISPR RNA&rsquo;s, surveils the cell and uses the RNA to home to the sequence with which it shares complementarity. Once it homes in, the CRISPR-Cas system scans for a motif called a PAM motif, which must be present at the target of interest. If the PAM motif is found, the CRISPR-Cas complex binds to the DNA and induces what&rsquo;s called a double stranded break (DSB). Once the break has been created, the cellular machinery of the invading pathogen stitches the broken area up, but without the sequence that had been cut. If the sequence coded for a protein that increased the pathogenicity of the invader, the invader is now <em>disabled</em>.&nbsp;</p>
        <h1>CRISPR-Cas: What can scientists do with it?&nbsp;</h1>
        <p>This very process can be exploited by scientists to edit the gene of their choice. The CRISPR RNA is around 23-55 nucleotides, and homes in to a sequence with perfect complementarity to induce a DSB. If the guide CRISPR RNA is engineered to reach the gene of interest, a DSB can be artificially induced anywhere. Techniques such as homology directed repair (HDR) also allow the cell to introduce a new sequence of interest at the location of a DSB. So, to modify sequence A into sequence B, the CRISPR-Cas system must be introduced into a cell, with the RNA sharing complementarity with A. Once the break is introduced, a HDR is initiated with sequence B, to make sequence B take A&rsquo;s place in the genome. If the intent is to simply delete a gene, sometimes referred to as a <em>knock-out</em>, an alternate pathway referred to as non- homologous end joining is utilised. However, it is important to note that the sequences that are cut are usually very small, and the process must be tweaked to remove entire genes.&nbsp;</p>
        <p>There are other ways CRISPR-Cas can be exploited. Since they recognise highly specific regions on the genome, they can also be used to map the structure of chromosomes in the relaxed state. They can also be used to make <em>epigenetic </em>modifications, modifications made directly to the genome by adding a structure to inhibit/ induce the gene.</p>
        <img src='../images/blog/blog 6 fig 4.png' height="200px"style="margin-left: 25%;"> 
        <p>Fig. 4: What to do after a DSB is induced.</p>
        <p>The advantage that CRISPR has over all its predecessors is specificity and accuracy. The Cas9 protein will not bind to any region that doesn&rsquo;t have both perfect complementarity to the RNA sequence it holds and the PAM motif mentioned above. While the latter might seem like a disadvantage, scientists seem to be on the verge of developing workarounds to that limitation.&nbsp;</p>
        <h1>Where do we go from here?&nbsp;</h1>
        <p>CRISPR-Cas9 has been revolutionary in academia and the impact has already started to extend to the clinical world. Human trials to treat genetic disorders are underway, with one woman in the United States being treated of sickle-cell anaemia using CRISPR therapy. While clinical applications progress, CRISPR has also given researchers vast potential to understand complex genes across various organisms and a quasi-regulatory mechanism, which can be exploited to understand the inner workings of various organisms. CRISPR is, hence, an invaluable tool in the toolkit of the microbiologist, the molecular biologist and the geneticist and most anyone trying to understand life.&nbsp;</p>
        <p>But as we go down this path utilising one of the most powerful tools ever innovated, it is also the pertinent that we have discussions in the public forum on the ethics of different kinds of gene editing, and how legislation would be framed in this regard. A review of the legality worldwide, conducted in 2016, characterised large portions of genome editing policy worldwide as &lsquo;vague&rsquo; and not sufficiently comprehensive. To truly utilise these amazing tools, we must make sure we account for the deleterious effects they could have when placed in the wrong hands, and ensure, to the greatest extent, that the tools are not misused as we move forward into a new frontier of human history.</p>
  
</div>
 
    </div>
    <div class="rightcolumn">
      <div class="card1">
        <h2>Written by</h2>
        <div class="name" style="height:100px;">Sai Guha C.</div>
       
      </div>
      <div class="card1">
        <h3>Reference</h3>
        <div class="name"><p>Doudna, J. A. &amp; Charpentier, E. The new frontier of genome engineering with CRISPR-&nbsp;Cas9. Science 346, (2014). doi: <a href="https://doi.org/10.1126/science.1258096" style="color: purple;">10.1126/science.1258096</a>.</p></div><br>
        
      </div>
      <div class="card1">
        <h3>Share:<a href='http://twitter.com/share?text=Synthetic%20Biology%20Goes%20Cell-Free&url=https://biotechstudentassociation.github.io/SyntheticBiologyGoesCellFree/'>Twitter</a>,<a href='https://www.facebook.com/sharer.php?u=https://biotechstudentassociation.github.io/SyntheticBiologyGoesCellFree/'>Facebook</a></h3>
        
      </div>
    </div>
  </div>
  <br>
<div class="footer">
  <p>Follow Us &emsp; <a href="https://www.instagram.com/bsaiitm/" target="_blank" ><i class="fa fa-instagram" style="font-size:36px"></i></a>&emsp;<a href="https://www.facebook.com/biotechresearchclubiitmadras" target="_blank" ><i class="fa fa-facebook-square" style="font-size:36px"></i></a>&emsp;<a href="mailto:biotechresearchclubiitm@gmail.com" target="_blank"><i class="fa fa-envelope" style="font-size:36px"></i></a>&emsp;<a href="https://chat.whatsapp.com/AnLJuBtdjXP5ltT8dtlLrR" target="_blank"><i class="fa fa-whatsapp" style="font-size:36px"></i></a></p>
 
</div>
<script src="../navbar.js">
    
</script>


</body>
</html>
